sdk/storage/test/test_base.cpp

```diff
@@ -92,22 +92,34 @@ namespace Azure { namespace Storage { namespace Test {
     return charset[distribution(random_generator)];
   }
 
-  std::string RandomString()
+  std::string RandomString(size_t size)
   {
     std::string str;
-    str.resize(10);
+    str.resize(size);
     std::generate(str.begin(), str.end(), random_char);
     return str;
   }
 
-  std::string LowercaseRandomString()
+  std::string LowercaseRandomString(size_t size)
   {
-    auto str = RandomString();
+    auto str = RandomString(size);
     std::transform(
         str.begin(), str.end(), str.begin(), [](unsigned char c) { return char(std::tolower(c)); });
     return str;
   }
 
+  std::map<std::string, std::string> RandomMetadata(size_t size)
+  {
+    std::map<std::string, std::string> result;
+    for (unsigned i = 0; i < size; ++i)
+    {
+      // TODO: Use mixed casing after Azure::Core lower cases the headers.
+      // Metadata keys cannot start with a number.
+      result.insert("m" + LowercaseRandomString(5), LowercaseRandomString(5));
```

Comment:
Value should not be all-lowercase

Reply:
This is changed to call `RandomString` instead for metadata value.